module/batch-action/src/Domain/Command/EndBatchActionProcessCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\BatchAction\Domain\Command;
+
+use Ergonode\BatchAction\Domain\Entity\BatchActionId;
+use Ergonode\BatchAction\Domain\ValueObject\BatchActionType;
+use Ergonode\SharedKernel\Domain\DomainCommandInterface;
+
+class EndBatchActionProcessCommand implements DomainCommandInterface
+{
+    private BatchActionId $id;
+
+    private BatchActionType $type;
+
+    public function __construct(BatchActionId $id, BatchActionType $type)
```

Comment:
BatchActionType is redundant you can fetch this information in handler by id.

In addition, if you have both values, there is a possibility that someone will use this command with an id of a different type than the type specified in the command, which will be an error